### ISSUE TITLE
Go to homepage in 404 pages

### ISSUE THREAD

**Issue by manorit2001** [XploreX/media-server#31]
**Is your feature request related to a problem? Please describe.**
TV history usually has stale urls for the media-server due to previous media-location and leads to 404 error response. Manually changing url in the TV to go the the home directory is not very user friendly and a very tedious task.

**Describe the solution you'd like**
A go to homepage button in 404 pages so that navigating to home directory due to these issues is easy.

**Comment by parth-07** [comment]
How about on 404 pages we display something like, "This content does not exist, would you like to go to _home directory_ ". This will be more accurate and expected behaviour, and more easy to debug for an uninitiated user.

**Comment by manorit2001** [comment]
I found a beautiful template, please check

**Comment by manorit2001** [comment]
**Here is a GIF showing the template**
![404-page](https://user-images.githubusercontent.com/40723245/103169791-f6d4c200-4864-11eb-846e-8724bacb419b.gif)

**Comment by parth-07** [comment]
This template seems really good.

**Comment by manorit2001** [comment]
Cool, then I am closing this issue

**Comment by manorit2001** [comment]
**Is your feature request related to a problem? Please describe.**
It would be nice if users got redirected to the home page automatically(similar to [youtube.com/tv](https://www.youtube.com/tv))

**Describe the solution you'd like**
How about adding a countdown and redirecting users back to the home page

**Screenshot**
The image shows 20s countdown but this is too much and we can reduce it down to 10.
![timer](https://user-images.githubusercontent.com/40723245/103303669-8e384180-4a2c-11eb-8e12-ed338432e938.gif)

**References**
https://css-tricks.com/how-to-create-an-animated-countdown-timer-with-html-css-and-javascript/

**Comment by parth-07** [comment]
Yeah, it's definitely a good idea. I think timer of 5s would be ideal, what do you think ?

**Comment by manorit2001** [comment]
5s sounds ideal to me also. I had suggested a timer of 10s by looking at [youtube.com/tv](https://www.youtube.com/tv), though am not sure why they have given this much time in that page. Both the pages (404 page and youtube one) doesn't have anything except redirection in them.
